packages/user-interface/src/components/CasesList.tsx

```diff
@@ -57,9 +59,10 @@ const CasesList = ({
   completed,
   showHeader,
 }: CasesListProps) => {
+  const { paths } = useOutletContext<RouterOutletContext>();
   const { data, loading, error, refetch } = useGetZakenQuery();
   const intl = useIntl();
-  const getCaseUrl = (id: string) => `/zaken/zaak?id=${id}`;
+  const getCaseUrl = (id: string) => paths.case.replace(":id", id);
```

Comment:
Haha dit is ook wel handig. Alleen moet je dan op verschillende plekken replace doen. Dit zou je dus ook in de paths const kunnen plaatsen.